packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -99,7 +91,7 @@ contract Multisig is PluginUUPSUpgradeable, ProposalUpgradeable, Addresslist {
     /// @notice Thrown if the minimal approvals value is out of bounds (less than 1 or greater than the number of members in the address list).
     /// @param limit The maximal value.
     /// @param actual The actual value.
-    error MinApprovalsOutOfBounds(uint256 limit, uint256 actual);
+    error MinApprovalsOutOfBounds(uint256 limit, uint16 actual);
```

Comment:
Why not
`error MinApprovalsOutOfBounds(uint16 limit, uint16 actual);` or
`error MinApprovalsOutOfBounds(uint32 limit, uint32 actual);`?